packages/core/test/testData/getRegistry.tsx

```diff
@@ -0,0 +1,23 @@
+import { createSchemaUtils, englishStringTranslator, Registry } from '@rjsf/utils';
+import getDefaultRegistry from '../../src/getDefaultRegistry';
+import validator from '@rjsf/validator-ajv8';
+
+export default function getRegistry(
```

Comment:
This is just a suggestion
```suggestion
export default function getTestRegistry(
```

Reply:
Good suggestion